### PR TITLE
[Examples] Update for Xcode 8 GM

### PR DESCRIPTION
**Pull request by DanToml**

### CODE REVIEW

**Comment by benasher44** [comment]
👍

**Comment by segiddins** [comment]
👍 once CI is green

**Comment by orta** [comment]
I re-ran the builds, they all fail on the same one still

```
Example: examples/TestInclusions
    Installing Pods
    Building scheme: TestInclusions
```

**Comment by benasher44** [comment]
Looks like you're hitting travis-ci/travis-ci#6422

**Comment by benasher44** [comment]
@DanToml I tried someone's workaround from travis-ci/travis-ci#6251, and this worked for me in another project. If you set the destination to iPhone SE (from the current iPhone 6s, it appears to fix the tests not running issue), that should hopefully fix it.

**Comment by orta** [comment]
I've pushed a commit to try running with iPhone SE - I'm not sure if there's a better way to do this via fourflusher - if there isn't then we can push /amend my commit removing the TODO

File: Rakefile [hunk 1]
```diff
@@ -281,9 +281,10 @@ begin
             when :ios
               require 'fourflusher'
               # Specifically build against the simulator SDK so we don't have to deal with code signing.
-              test_flag = (scheme_name.start_with? 'Test') ? 'test' : ''
-              destination = Fourflusher::SimControl.new.destination(:oldest, platform, target.deployment_target).join(" ")
-              execute_command "xcodebuild -workspace '#{workspace_path}' -scheme '#{scheme_name}' clean build #{test_flag} ONLY_ACTIVE_ARCH=NO #{destination}"
+              test_flag = scheme_name.start_with?('Test') ? 'test' : ''
+              # DO NOT MERGE
```
Comment (segiddins) [inline Rakefile]: ❌

**Comment by segiddins** [comment]
Is it just a bug with Travis or a bug in four-flusher?

**Comment by orta** [comment]
Xcode from the sound of the issues - travis doesn't seem to be running my commit

**Comment by segiddins** [comment]
@orta probably because of the merge conflict

**Comment by orta** [comment]
ah yeah, lets see then

**Comment by segiddins** [comment]
@orta please rebase instead of merging master

**Comment by DanToml** [comment]
Superseded by #6282

**Pull request closed without merging**